internal/policy/container/has_prohibited_labels.go

```diff
@@ -0,0 +1,65 @@
+package container
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/check"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/image"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/log"
+
+	"github.com/go-logr/logr"
+)
+
+var trademarkLabels = []string{"name", "vendor", "maintainer"}
+
+var _ check.Check = &HasProhibitedLabelsCheck{}
+
+type HasProhibitedLabelsCheck struct{}
```

Comment:
This naming is unclear to me. This check asserts that labels don't have prohibited value, which *today* translates into trademark violations. This check does *not* check that given label keys are missing, as the name might suggest.

Reply:
This has always been a problem with our naming, I can change it to `HasNoProhibitedLabelsCheck`

Reply:
Eh. I didn't catch that this was the old name. I don't love it, given what it's doing now, but I suppose I can't  say much given the old name was basically this.

Reply:
Up to you how you want to handle this.

Reply:
It might be time to do a review of our check naming? As @komish has stated, some of them are just not named in a way that is unambiguous.

Reply:
I changed this `HasNoProhibitedLabelsCheck`, I hope this helps.